src/forgot-password/data/sagas.js

```diff
@@ -20,9 +21,9 @@ export function* handleForgotPassword(action) {
     yield put(forgotPasswordSuccess(action.payload.email));
   } catch (e) {
     if (e.response && e.response.status === 403) {
-      yield put(forgotPasswordForbidden());
+      yield put(forgotPasswordForbidden(null));
     } else {
-      throw e;
+      yield put(forgotPasswordForbidden(INTERNAL_SERVER_ERROR));
```

Comment:
Why we are using forbidden action for 500 errors? It should have its own action and should utilities the `status` prop.